COMPATIBILITY.md

```diff
@@ -19,8 +19,8 @@ The below matrix shows the compatibility of the [`opensearch-net`](https://www.n
 | 1.3.1 | 1.0.0 |
 | 1.3.2 | 1.0.0 |
 | 1.3.3 | 1.0.0 |
-| 2.0.0 | 2.0.0 |
-| 2.0.1 | 2.0.0 |
+| 2.0.0 | 1.0.0 |
+| 2.0.1 | 1.0.0 |
 
 
 The client is also fully compatible with [`OpenDistro`](https://opendistro.github.io/for-elasticsearch/) 1.13.x.
```

Comment:
You can remove this line while you're at it

Reply:
Also, the "Upgrading" section below only applies to OpenSearch 2.0 client.  It's not relevant for a 1.x release.  

I'm not sure removing it is necessary, but you could prefix the section with `For OpenSearch 2.0 client release:`

Reply:
It is still compatible, customers might use client with existing OpenDistro clusters. We can't run tests on it.

Reply:
If we can't run tests, we shouldn't support it, and we can remove this bit of documentation.  Plus, we want to encourage people to migrate to opensearch.